install_config/configuring_openstack.adoc

```diff
@@ -40,7 +39,55 @@ Consult your OpenStack administrators for values of the `*OS_*` variables, which
 are commonly used in OpenStack configuration.
 
 [[openstack-configuring-masters]]
-== Configuring Masters
+== Configuring OpenStack for {product-title} Masters
```

Comment:
@tpoitras my complaint with this is mainly that in other sections (provider/platform docs) we structure the lay out of information like: 

```
Configuring PROVIDER for OpenShift Enterprise COMPONENT
    Configuring PROVIDER for OpenShift Enterprise with Ansible
    Manually Configuring PROVIDER for OpenShift Enterprise COMPONENT
    Manually Configuring PROVIDER for OpenShift Enterprise COMPONENT
```

Why would we not do the same for this provider/platform?